apps/explorer/src/components/Layout/PageLayout.tsx

```diff
@@ -39,14 +42,18 @@ export function PageLayout({ gradient, content, loading, isError }: PageLayoutPr
 	});
 	const isGradientVisible = !!gradient;
 	const renderNetworkDegradeBanner = network === Network.MAINNET && data?.degraded;
+	const headerRef = useRef<HTMLElement | null>(null);
+	const headerHeight = useElementHeight(headerRef, DEFAULT_HEADER_HEIGHT);
```

Comment:
Need to handle smaller breakpoints where the banner might become more than 1 line. It can go up to 3 lines and I do not want to hardcode it, in the event we add more things here or modify the text contents

Reply:
It will tick correctly since it will run when the ref gets updated the first time. I have screenshotted the console logging before when inputing default as 0

<img width="233" alt="Screenshot 2023-08-01 at 2 49 52 PM" src="https://github.com/MystenLabs/sui/assets/127577476/e138c66a-afe4-437b-b400-ff81335e11c8">